hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/GenericTestUtils.java

```diff
@@ -229,6 +229,31 @@ public static int uniqueSequenceId() {
     return sequence.incrementAndGet();
   }
 
+  /**
+   * Creates a directory for the data/logs of the unit test.
+   * It first delete the directory if it exists.
+   *
+   * @param testClass the unit test class.
+   * @param defaultTargetRootDir the directory where the class directory is
```

Comment:
This description is a little unclear.  Maybe default relative path to use if SYSPROP_TEST_DATA_DIR is not set?

Reply:
I'm actually not sure we need to pass a default here.  Everywhere we use it, it is just using DEFAULT_TEST_DATA_DIR.